packages/medusa/src/utils/flag-router.ts

```diff
@@ -0,0 +1,17 @@
+import { IFlagRouter } from "../types/feature-flags"
+
+export class FlagRouter implements IFlagRouter {
+  private flags: Record<string, boolean> = {}
+
+  constructor(flags: Record<string, boolean>) {
+    this.flags = flags
+  }
+
+  public featureIsEnabled(key: string): boolean {
```

Comment:
suggestion: Just a reminder of the far away comment on that 😂 `isFeatureEnabled`